.github/workflows/test.yml

```diff
@@ -74,3 +74,15 @@ jobs:
             bun = "1"
       - run: which bun
       - run: bun -v
+
+  with_settings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup mise
+        uses: ./
+        with:
+          settings: |
+            always_keep_download=true
```

Comment:
this should be an object

Reply:
(i would have loved that, but afaik you can't pass objects in github actions params, only multiline strings which ends up being `string[]`)